### PR TITLE
CLDR-17246 Muscogee locale data changes

### PR DESCRIPTION
**Pull request by i18ncraig**
CLDR-17246

- [x] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Review by srl295 — DISMISSED** [review]
lgtm

File: common/main/mus.xml [hunk 1]
```diff
@@ -89,13 +89,13 @@ CLDR data files are interpreted according to the LDML specification (http://unic
...
 						<dayWidth type="wide">
-							<day type="sun">Nettv Cako</day>
+							<day type="sun">Nettv'cako</day>
 							<day type="mon">Enhvteceskv</day>
 							<day type="tue">Enhvteceskv Enhvyvtke</day>
 							<day type="wed">Ennvrkvpv</day>
 							<day type="thu">Ennvrkvpv Enhvyvtke</day>
 							<day type="fri">Nak Okkoskv Nettv</day>
-							<day type="sat">Nettv Cakcuse</day>
+							<day type="sat">Nettv cak'cuse</day>
```
Comment (macchiati) [inline common/main/mus.xml:98]: This appears to be inconsistent titlecasing. Can you check?
Nak Okkoskv Nettv
vs
Nettv cak'cuse

**Comment by i18ncraig** [comment]
Checking with Muscogee language folks on the casing differences...

**Comment by srl295** [comment]
@i18ncraig per today's meeting, you said you would fix the casing and then I can re-review

**Comment by i18ncraig** [comment]
I have edited the pull request -- adding comments that per the Muscogee
Nation language program folks, days of the week should have init caps on
each word. Fixed accordingly. It has been a while since I've done PR edit,
so hope all is good.

**Review by srl295 — APPROVED** [review]

**Pull request merged**